src/main/java/uk/gov/hmcts/darts/common/datamanagement/helper/StorageOrderHelper.java

```diff
@@ -0,0 +1,40 @@
+package uk.gov.hmcts.darts.common.datamanagement.helper;
+
+import jakarta.annotation.PostConstruct;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Configuration;
+import uk.gov.hmcts.darts.common.datamanagement.enums.DatastoreContainerType;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Configuration
+@Slf4j
+@RequiredArgsConstructor
+public class StorageOrderHelper {
+
+    @Value("#{'${darts.storage.search.order}'.split(',')}")
```

Comment:
Please can you look to use `DetsDataManagementConfiguration configuration` here and maintain it with the new application property, so that we can remove the use of `@Value` as I thought it was our best practice not to use them.

I feel that the application config should support List easily without the need for messing about splitting stuff - please can you refer to other ConfigurationProperties we have like `darts.transcription.allowed-extensions`.  See TranscriptionConfigurationProperties#allowedExtensions

Reply:
This is news to me, in fact i was chatting to Tony who said he prefers to use @Value.  Where is this mentioned for best practice?

Reply:
You're right, I've looked up our confluence page and don't see any mention of it there.
Anyway, I'll leave the rest of my comment there for your consideration.

Reply:
i'd still prefer to leave it as is, as its quite clean, and it will only get loaded once at startup.